hedy.py

```diff
@@ -157,17 +157,48 @@ def promote_types(types, rules):
 
 character_skulpt_cannot_parse = re.compile('[^a-zA-Z0-9_]')
 
+
+def get_list_keywords(commands, to_lang):
```

Comment:
@Felienne This function was causing a circular import so I moved it to Hedy.py, with permission of @teundemast  
It does bring up issue #1482 since not everyone knows that the boolean has to be set to True for the tests to run

Reply:
I see! Can we merge this now (it seems the tests are passing?) or do we have to wait until #1482 is fixed?

Reply:
I can add it in this branch, since i need to update the textwrap dedent anyway :)

Reply:
I mean add Issue #1482

Reply:
Yeah I got that, let's do it here than so all is in order at once.

Thanks!